docs/platform_concepts/engagement/engage_faqs.md

```diff
@@ -54,6 +54,17 @@ You can use [User360 events](https://docs.yellow.ai/docs/platform_concepts/studi
 
 </details>
 
+
+<details>
+<summary>Getting this error when creating a WhatsApp template `Cannot read property 'wabaID' of null`</summary>
```

Comment:
Hey just a suggestion, we can create a new section such as **Troubleshooting guide** and error and solutions under that so that when users encounter a specific error, they can refer to that section.